parsl/data_provider/zip.py

```diff
@@ -76,6 +82,20 @@ def stage_out(self, dm, executor, file, parent_fut):
         app_fut = stage_out_app(zip_path, inside_path, working_dir, inputs=[file], _parsl_staging_inhibit=True, parent_fut=parent_fut)
         return app_fut
 
+    def stage_in(self, dm, executor, file, parent_fut):
+        assert file.scheme == 'zip'
+
+        zip_path, inside_path = zip_path_split(file.path)
+
+        working_dir = dm.dfk.executors[executor].working_dir
+
+        if working_dir:
+            file.local_path = os.path.join(working_dir, inside_path)
+
+        stage_in_app = _zip_stage_in_app(dm)
+        app_fut = stage_in_app(zip_path, inside_path, working_dir, outputs=[file], _parsl_staging_inhibit=True, parent_fut=parent_fut)
+        return app_fut._outputs[0]
+
 
 def _zip_stage_out(zip_file, inside_path, working_dir, parent_fut=None, inputs=[], _parsl_staging_inhibit=True):
     file = inputs[0]
```

Comment:
For both `_zip_stage_in` and `_zip_stage_out`, are these interfaces set by another driver?  I'm observing that in _these_ internal methods, the following arguments don't currently appear to be utilized:
- `working_dir`
- `parent_fut`
- `_parsl_staging_inhibit`